var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -368,8 +368,8 @@ def _setup_dependent_env(self, env, dependent_spec, deptypes):
         if perl_lib_dirs:
             perl_lib_path = ":".join(perl_lib_dirs)
             env.prepend_path("PERL5LIB", perl_lib_path)
-        if is_windows:
-            env.append_path("PATH", self.prefix.bin)
+            if is_windows:
+                env.append_path("PATH", self.prefix.bin)
```

Comment:
I see this is moving this, but I'm surprised this is needed: we should already be adding `<prefix>/bin/` to the `PATH` automatically. Does the installation on windows work without this?